### PR TITLE
Add more permissions nodes for better configuration

### PR DESCRIPTION
**Pull request by Intybyte**

### CODE REVIEW

File: bukkit/src/main/resources/plugin.yml [hunk 1]
```diff
@@ -9,28 +9,122 @@ api-version: '1.13'
...
+            auraskills.skill.endurance: true
+            auraskills.skill.agility: true
+            auraskills.skill.alchemy: true
+            auraskills.skill.enchanting: true
+            auraskills.skill.sorcery: true
+            auraskills.skill.healing: true
+            auraskills.skill.forging: true
         default: true
+
+    auraskills.command.itemmaker:
```
Comment (Archy-X) [inline bukkit/src/main/resources/plugin.yml:55]: Rename this to auraskills.command.item.*

File: bukkit/src/main/resources/plugin.yml [hunk 2]
```diff
@@ -9,28 +9,122 @@ api-version: '1.13'
...
+        description: All permissions related to the creation of items using Auraskills modifier
```
Comment (Archy-X) [inline bukkit/src/main/resources/plugin.yml:56]: Change to "AuraSkills" instead of "Auraskills"

File: bukkit/src/main/resources/plugin.yml [hunk 3]
```diff
@@ -9,28 +9,122 @@ api-version: '1.13'
...
+            auraskills.command.item.multiplier: true
+            auraskills.command.item.register: true
+            auraskills.command.item.give: true
+            auraskills.command.item.ignore: true
+        default: false
+
+    auraskills.command.admin:
+        description: Permission to use all admin/op commands
+        children:
+            auraskills.command.itemmaker: true
```
Comment (Archy-X) [inline bukkit/src/main/resources/plugin.yml:72]: Change to match renamed permission

**Review by Archy-X — CHANGES_REQUESTED** [review]
Some minor renaming

File: bukkit/src/main/resources/plugin.yml (hunk 1 above)
Comment (Intybyte) [reply bukkit/src/main/resources/plugin.yml:55]: then i am putting these

```yml
auraskills.command.armor.modifier: true
auraskills.command.armor.requirement: true
auraskills.command.armor.multiplier: true
```

into their own `auraskills.command.armor.*`  for clarity

**Review by Archy-X — APPROVED** [review]

**Pull request merged**